sentry/__init__.py

```diff
@@ -3,80 +3,118 @@
 
 import logging
 
+import odoo.http
```

Comment:
Move all logic from `__init__.py` file to [hooks.py](https://github.com/OCA/maintainer-tools/blob/master/template/module/hooks.py)

And import hooks like:
 - https://github.com/OCA/maintainer-tools/blob/f1b559574648c01558b543e7550f5b9be7c92784/template/module/__init__.py#L7

Reply:
Done